### PR TITLE
Working UI with snowbridge

### PR DESCRIPTION
**Pull request by sumitsnk**
Some changes have been made so that the UI may be used with snow-bridge main.

### CODE REVIEW

File: src/config-local.tsx [hunk 1]
```diff
@@ -57,5 +57,6 @@ export default {
     { "parachainName":'Bifrost (coming soon...)', "parachainId": 0,'isDisabled':true },
     { "parachainName":'Kusama (coming soon...)', "parachainId": 0, 'isDisabled':true },
     { "parachainName":'Snowbridge (coming soon...)', "parachainId": 0,'isDisabled':true }
-  ]
+  ],
+  TRANSACTION_FEES:'0'
```
Comment (alistair-singh) [inline src/config-local.tsx:61]: Fees would be a different per parachain. So it needs to be part of `PARACHAIN_LIST` lists parachain data.

File: src/config-staging.tsx [hunk 2]
```diff
@@ -59,5 +59,6 @@ export default {
     { "parachainName":'Bifrost (coming soon...)', "parachainId": 0,'isDisabled':true },
     { "parachainName":'Kusama (coming soon...)', "parachainId": 0, 'isDisabled':true },
     { "parachainName":'Snowbridge (coming soon...)', "parachainId": 0,'isDisabled':true }
-  ]
+  ],
+  TRANSACTION_FEES:'0'
```
Comment (alistair-singh) [inline src/config-staging.tsx:63]: Same as above.

File: src/net/eth.ts [hunk 3]
```diff
@@ -290,7 +291,7 @@ export default class Eth extends Api {
       // call ether contract for ether
       if (isEther(asset)) {
         return ethContract.methods
-          .lock(polkadotRecipientBytes, channelId, parachainId)
+          .lock(polkadotRecipientBytes, channelId, parachainId,TRANSACTION_FEES)
```
Comment (alistair-singh) [inline src/net/eth.ts:294]: You would have to plumb the data down from the list the same way we do `parachainId`.

File: src/config-local.tsx (hunk 1 above)
Comment (sumitsnk) [reply src/config-local.tsx]: As suggested, I've updated the changes.

File: src/config-staging.tsx (hunk 2 above)
Comment (sumitsnk) [reply src/config-staging.tsx]: As suggested, I've updated the changes.

File: src/net/eth.ts (hunk 3 above)
Comment (sumitsnk) [reply src/net/eth.ts]: As suggested, I've updated the changes.

**Review by alistair-singh — APPROVED** [review]

**Pull request merged**